dazel.py

```diff
@@ -320,13 +324,14 @@ def _run_container(self):
         logger.info("Starting docker container '%s'..." % self.instance_name)
         command = "%s stop %s >/dev/null 2>&1 ; " % (self.docker_command, self.instance_name)
         command += "%s rm %s >/dev/null 2>&1 ; " % (self.docker_command, self.instance_name)
-        command += "%s run -id --name=%s %s %s %s %s %s %s%s %s" % (
+        command += "%s run -id --name=%s %s %s %s %s %s %s %s%s %s" % (
```

Comment:
I think there's a space missing between the %s-s.

Reply:
I added `%s` before the connected one as is above, the git-diff looks a little funky though.